README.md

```diff
@@ -31,37 +36,255 @@ npm install @uploadcare/upload-client --save
 
 ## Usage
 
+### High Level API
+
+To get access to High Level API you need to create 
+an instance of `UploadClient` and provide settings to him:
+
 ```javascript
 import UploadClient from '@uploadcare/upload-client'
 import {FileFromEnum} from '@uploadcare/upload-client'
 
 const client = new UploadClient({publicKey: 'YOUR_PUBLIC_KEY'})
+```
 
-client.api.request({path: 'info', query})
-  .then(response => console.log(response.data))
+After that you can upload file from binary data:
 
-client.api.info(uuid)
-  .then(data => console.log(data.is_image))
+```javascript
+const fileUpload = client.fileFrom(FileFromEnum.Object, fileData)
 
-const directUpload = client.api.base(fileData)
+fileUpload
+  .then(file => console.log(file.uuid))
+```
 
-directUpload
-  .then(data => console.log(data.file))
+Or from URL with file:
 
-directUpload.onProgress = (progressEvent) => console.log(progressEvent.loaded / progressEvent.total)
+```javascript
+const fileURL = 'https://example.com/file.jpg'
+const fileUpload = client.fileFrom(FileFromEnum.URL, fileURL)
 
-const fileUpload = client.fileFrom(FileFromEnum.Object, fileData)
+fileUpload
+  .then(file => console.log(file.uuid))
+```
+
+Or even from `uploaded` before to Uploadcare file:
```

Comment:
You can also use the `fileFrom` method to get previously uploaded files via their UUIDs,